filters/openpolicyagent/openpolicyagent.go

```diff
@@ -395,6 +398,8 @@ func interpolateConfigTemplate(configTemplate []byte, bundleName string) ([]byte
 // new returns a new OPA object.
 func (registry *OpenPolicyAgentRegistry) new(store storage.Store, configBytes []byte, instanceConfig OpenPolicyAgentInstanceConfig, filterName string, bundleName string, maxBodyBytes int64, bodyReadBufferSize int64) (*OpenPolicyAgentInstance, error) {
 	id := uuid.New().String()
+	uniqueIDGenerator, _ := flowid.NewStandardGenerator(32)
```

Comment:
https://github.com/zalando/skipper/pull/2926/files#r1508936434

Reply:
Returned the error from here. We have the option of letting it go with the existed flow if IDGenerator creation failed for some reason. Went against it, as then we are silently falling back to a sub-optimal flow.